frontend/src/components/CircularMenu/CircularMenu.js

```diff
@@ -0,0 +1,82 @@
+import React, { useState } from "react";
+import "./CircularMenu.scss";
+import PropTypes from "prop-types";
+
+const CircularMenu = ({
+  itemsAmount,
+  x,
+  y,
+  row,
+  column,
+  updateBoard,
+  hideMenu
+}) => {
+  const tilesArray = [1, 2, 3, 4, 5, 6, 7, 8, 9];
+  const [opened, setOpened] = useState(false);
+
+  const displayMenu = () => {
+    if (opened) {
+      hideMenu();
+      setOpened(false);
+    } else {
+      document.getElementById("circle").classList.toggle("open");
```

Comment:
Direct dom manipulation most of the times mean you are using react wrong. This logic should be in the component itself like this:

```<div id="circle" className={`circle ${opened ? '' : 'open'}`}>```